.travis.yml

```diff
@@ -1,12 +1,5 @@
 language: php
 
-php:
```

Comment:
Travis allows to use both the build-matrix (advanced) and/or the simple `php` versions list.
But if you already use the matrix its cleaner to not use a list also (it's less confusing :+1:  ).